raiden/tests/scenarios/mfee2_proportional_fees.yaml

```diff
@@ -72,8 +73,8 @@ scenario:
             - assert: {from: 0, to: 1, total_deposit: 100_000, balance: 89_789, state: "opened"}
             - assert: {from: 1, to: 0, total_deposit: 0, balance: 10_211, state: "opened"}
 
-            - assert: {from: 1, to: 2, total_deposit: 100_000, balance: 89_890, state: "opened"}
-            - assert: {from: 2, to: 1, total_deposit: 0, balance: 10_110, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 100_000, balance: 89_887, allow_balance_error: 1, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 0, balance: 10_113, allow_balance_error: 1, state: "opened"}
 
-            - assert: {from: 2, to: 3, total_deposit: 100_000, balance: 89_990, state: "opened"}
-            - assert: {from: 3, to: 2, total_deposit: 0, balance: 10_010, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 100_000, balance: 89_988, state: "opened"}
```

Comment:
should we add `allow_balance_error: 1` here, too? Even if not needed but out of consistency.

Reply:
It should not be needed, because we only exactly hit the boundary for the mediator and neither for the initiator nor the target.